zipline/pipeline/factors/basic.py

```diff
@@ -54,6 +54,28 @@ def compute(self, today, assets, out, close):
         out[:] = (close[-1] - close[0]) / close[0]
 
 
+class PercentChange(CustomFactor):
+    """
+    Calculates the percent change in the input value over the given window_length.
+    Note: Uses the formula "new - old / abs(old)" to handle negative input values
+
+    **Default Inputs:** None
```

Comment:
Looks like other factors mention `**Default Window Length**: None`, might be worth adding that here too.